File: programming-basics-python/03b-conditional-statements-advanced-exercise/01_cinema.py
type_screening = input()
number_rows = int(input())
number_colums = int(input())
income = 0

if type_screening == "Premiere":
    income = number_colums * number_rows * 12

elif type_screening == "Normal":
    income = number_colums * number_rows * 7.5

elif type_screening == "Discount":
    income = number_colums * number_rows * 5

print(f"{income:.2f} leva")

File: a1-programming-basics-javascript/03b-conditional-statements-advanced-exercise/01-cinema.js
function cinema(input) {
  let screeningType = input[0];
  let numberOfRows = Number(input[1]);
  let numberOfCols = Number(input[2]);
  let pricerPerTicket = 0;

  if (screeningType === "Premiere") {
    pricerPerTicket = 12.0;
  } else if (screeningType === "Normal") {
    pricerPerTicket = 7.5;
  } else {
    pricerPerTicket = 5.0;
  }

  income = (numberOfRows * numberOfCols * pricerPerTicket).toFixed(2);
  console.log(`${income} leva`);
}
